### DISCUSSION TITLE
Newtonian Cosmology with modified gravity law?

### DISCUSSION THREAD

**Discussion by genotics** [a-cosmology-group/acg#196] [Cosmology Models]
Hello friends,
With this note I propose to seed a discussion of alternate cosmology with an old article by Peter F. Browne that appeared in Apeiron in 1994. Peter, who passed away a decade ago, had long been active in the field during his career at UMIST in Manchester. In fact, the earliest publication of his that I have able to locate was  an article on a tired-light deSitter cosmology that appeared in Nature in 1964. Of course the 1994 article too was written long before dark topics became the rage in cosmology; yet one proposition in that article seems to foreshadow the darkness to come: "Electromagnetic energy density in the external field of an electron provides gravitational mass density which is the source of a gravitational field. The gravitational field, in turn, provides energy density which makes a further contribution to the source of the same field (i.e. a self-source term)."  
From this he deduces that  "A universe, therefore, behaves as a black hole for both an internal and an external observer. The black
hole horizon at r = R severs connection between regions r < R and r > R, in which sense a universe becomes a perfectly isolated system in an infinite Cosmos."
He defines an aether fluid, whose velocity is assumed non-uniform in the presence of a gravitational field. Thus there are two Doppler shifts: that "relative to the Euclidean frame a normal Doppler shift, and that relative to the aether fluid a generalized Doppler shift."  ? If the source is at rest relative to the Euclidean frame, it still has motion relative to the aether fluid, and the Doppler shift due to this motion is called a “gravitational redshift.”"
I find it difficult to wrap my mind around this dual Doppler shift concept, and yet it seems eerily akin to the notion of a redshift due to the generalized relativity I tinkered with back in the early 80s.  Moreover, I find a striking parallel between this generalized Doppler shift and the cosmic drag Amitabha deduced from the inertial induction hypothesis he put forward about the same time.
If any members are tempted to investigate the work of Peter and Amitabha as a possible foundation for a more robust cosmological (redshift) model, I would be happy to provide the references to which I have access. If you would like to consult articles in the Apeiron archive, please be forewarned that fonts have not been embedded in many of the PDF files, partially through oversight, but actually many files date from a time when I was producing paper originals for the printer.  There is a partial workaround: you can install the Mathtype editor in a Google Workspace , and  then open the archived PDFs in the same browser. Most of the Math fonts will appear correctly, with some exceptions, such as Greek characters. If anyone finds a more complete solution, please share with the rest of us. Here are the links to the article by Peter and one by Amitabha.
<http://redshift.vif.com/JournalFiles/Pre2001/V00NO20PDF/Nr20bro.pdf>
<http://redshift.vif.com/JournalFiles/Pre2001/V02no2PDF/V02N2GHO.PDF>

Here is the link to install Mathtype to Google Workspace: <https://store.wiris.com/en/products/mathtype/download>
I would be happy to provide clarifications if I am able. Cheers, Roy

**Comment by ExpEarth** [comment]
Hi Roy.  As the only one here (I think) to also be supporting a black hole universe, I feel a sort of obligation to make a comment or two.  What I have to offer comes from just my own perspective and is not based on too close a reading.  I can't follow the paper well at all and this is only partially to do with the distortions in the equations.

I think Browne gets into trouble right from the start in linking this to the zero-point fields.  While I used to like the ZPF too, and submitted a paper on that once (rejected), I have recently come to the view that they are not at all necessary.  It is possible to come up with a theory of gravity based only on the CMB, provided that we use the 'hotter' CMB found at remote distances from us.  The main evidence for the ZPF was considered to be the Casimir effect and even that has now come into question.  My position now would be that the ZPF are purely fictitious.

Concerning the problems in the equations in that paper, and other aspects of the Apeiron archive, have you given a thought to how to preserve this archive in a clearer, more secure format?  Maybe you (or someone) could apply for a grant to do this.  It's really too valuable to be as inaccessible as it seems to be.  The quality of those papers is generally far greater than what can be found in the non-mainstream journals today.

Matt

**Reply by HanDeBruijn** [reply]
>As the only one here (I think) to also be supporting a black hole universe [ .. ]

The only one here? Are you sure? Read [this comment](https://github.com/orgs/a-cosmology-group/discussions/127#discussioncomment-6789591) and [this one](https://github.com/orgs/a-cosmology-group/discussions/127#discussioncomment-6812985) too. And do us a favour.

**Reply by ExpEarth** [reply]
@HanDeBruijn 

Okay, I checked your posts there.  I think you are saying, more or less, that the universe is something like a black hole, but you prefer not to use that term.   The earlier authors called these objects dark stars.  The term black hole does imply a singularity and I object to that too.  I argue that a singularity is avoided due to what I call the Hubble luminosity.  I think you also have the mass of the black hole universe evenly distributed, whereas I have most of it in a plasma shell which encloses it.  Otherwise we could be on the same path!

**Reply by HanDeBruijn** [reply]
@ExpEarth.

>The term black hole does imply a singularity [ .. ]

Only in terms of general relativity **Theory**. Moreover it only seems to hold for the [non-rotating](https://github.com/orgs/a-cosmology-group/discussions/144#discussioncomment-7777551) specimen of the kind.
The original (Newtonian) meaning of _Black Hole_ is _a body from which no light can escape_. Indeed:
>The earlier authors called these objects dark stars [ .. ]

There are two flavours, the Einstein (GR) one and the Newton (+SR) one. Let $R$ be the radius and $M$ be the mass of a (celestial) body, then we have the following condition for becoming a Black Hole/Dark Star (where $c=$ lightspeed and $G=$ gravitational constant):

$$
\begin{matrix}
\mbox{Einstein} & \mbox{Newton} \\
R \le \frac{2GM}{c^2} & R \le \frac{GM}{c^2}
\end{matrix}
$$

>I argue that a singularity is avoided due to what I call the Hubble luminosity.

I've looked it up and found something in [Wikipedia](https://en.wikipedia.org/wiki/Reflection_nebula):
>the **Hubble luminosity** law for reflection nebulae, which makes a relationship between the angular size ($R$) of the nebula [i.e. **not** a galaxy] and the apparent magnitude ($m$) of the associated star:

$$
5\log(R) = - m + k
$$

>where $k$ is a constant that depends on the sensitivity of the measurement.

The only other reference I could find is [_Does the Hubble Redshift Flip Photons and Gravitons?_](https://arxiv.org/ftp/arxiv/papers/1111/1111.2331.pdf). But we already know that one, don't we? So it seems that your "Hubble luminosity" is not the one as commonly understood by the scientific community. And I wonder if it exists.

**Reply by ExpEarth** [reply]
Thanks, Han.  It's interesting that the radius for the Einstein black hole is twice as large as the Newtonian dark star.  In the general relativity black hole there is a specific relationship between the mass and the radius, _R = 2GM/c^2_ .  This actually defines the Schwarzschild radius.  I take it that the equivalent radius for the Newtonian black hole would be _GM/c^2_ ?  That could be handy for my model.

> The only other reference I could find is [Does the Hubble Redshift Flip Photons and Gravitons?](https://arxiv.org/ftp/arxiv/papers/1111/1111.2331.pdf). But we already know that one, don't we? So it seems that your "Hubble luminosity" is not the one as commonly understood by the scientific community. And I wonder if it exists.

It seems I chose a poor name for it and I didn't realize there was another way the term "Hubble luminosity" is being used.  I haven't found a better term for it yet.  If you or anyone has one I'd be happy to hear it!

As for its existence, you can bet on it!  Every day there is more evidence for it.  I do need to back up my claim with more research papers but haven't had the time to do so.  If anyone wanted to help me on this it would be great.  I'm not the best one for presenting data.  Here is a paper I did on the Hubble luminosity of white dwarfs.  They should be cool, but they are not!  Same as for neutron stars.

[http://redshift.vif.com/JournalFiles/V15NO4PDF/V15N4EDW.pdf](url)

**Reply by Francois-Zinserling** [reply]
Dear Roy @genotics 
For a moment I confused Peter F. Browne with Kevin Brown - author of mathpages.com. It was there that I had first encountered the push-gravity concept (although he is/was a critic) but I have not let go ...

_"A universe, therefore, behaves as a black hole for both an internal and an external observer. The black
hole horizon at r = R severs connection between regions r < R and r > R, in which sense a universe becomes a perfectly isolated system in an infinite Cosmos."_

This analogy makes sense up to the point that there appears to be a limit beyond which we cannot observe anything. An external observer could quite happily peer into our universe though, even if not all is visible to him. And from inside, if we were able to observe an object at distance just short <R, we should still see if it is interfered by another object at distance >R. In the case of a BH, Rs provides an absolute observation barrier between space at >Rs and <Rs, so the BH analogy of our universe is therefore limited in my view. 

_He defines an aether fluid, whose velocity is assumed non-uniform in the presence of a gravitational field. Thus there are two Doppler shifts: that "relative to the Euclidean frame a normal Doppler shift, and that relative to the aether fluid a generalized Doppler shift." ? If the source is at rest relative to the Euclidean frame, it still has motion relative to the aether fluid, and the Doppler shift due to this motion is called a “gravitational redshift.”_

For a gravitational redshift (as above) to be observable from all directions, the source cannot have a linear velocity relative to the aether fluid, else one side should be blueshifted. But if the aether fluid appears to be flowing into the mass, (his 'non-uniform velocity') then all sides are redshifted. This is also akin to Daniel Faccio’s representation of ‘space is sucked into earth’ in his river model. Which further, if you replace Fatio and Le Sage's 'corpuscles' with a fluid/dynamic (flowing) aether, presents a similar model. Redshift all round is then a conclusion.
By extension, if there is an 'inflow' or a case of 'outflow<inflow' of aether in proximity of masses, then Browne's Euclidian frame and aether fluid frames (edit: and gravitational field) can be united as one aether.

I look forward to reading your references and the rest of this discussion.

**Comment by genotics** [comment]
Hi Matt,
Thank you for your kind comments. Certainly something will need to be done about the missing math fonts, and I will see if I can cobble together a solution with the materials at hand first. Concerning ZPF, the variety of definitions available suggests that not everyone is talking about the same thing when alluding to it. In Browne's case, he identifies it with a fluid aether, which is distinct from CMB in that velocity cannot be measured relative to it.  Beyond that point, the physics gets pretty involved, but suffice it to say that the aether is the seat of gravity for him, as it was for Adolphe.
Is everyone aware of the video recently posted by Alex Unzicker on The Most Fundamental Problem of Gravity is Solved? It reports on recent work by one Jonathan Fay. See <https://youtu.be/BpQ0T7rDWm0?si=sk5ZEPj1wiPLZFsq> I would be interested in knowing everyone's thoughts.

**Reply by Francois-Zinserling** [reply]
Dear Roy @genotics 
Again I seem to disagree. I am aware of the Machian principle, describing inertia as a result of total gravity from all distant objects in the universe(simplisticly said). It still fails to describe what gravity is though, and it fails to describe how interaction with distant masses produce an inertial mass M, which is exactly the same mass M in local forced accelerations - as if the distant universe is controlling local actions too. 

In the above video around ~14min, Unzicker implies that a rotating universe will draw a curve in Newton's bucket. This is not true and easily defeated. In a different post, @RedshiftDrift (Louis Marmet) explained that the gravity from a very large (infinite wide) sheet would be the same everywhere you go. By extension, if the sheet were moving, up, down or even rotating, nothing would change. The same could be inferred (and proven) for a very large homogenous shell representing the distant masses of the universe. If it starts rotating, it will not change local conditions. A further argument I can offer is that a rotating magnet does not create a magnetic field, inward, or outward, which has been tested and proven. The same must be true for a rotating mass. (except for small relativistic effects expected, as found in GR)

**Reply by HanDeBruijn** [reply]
@Francois-Zinserling .

Me too: I have not been able to (mathematically) reproduce Unzicker's shocking discoveries.
>A further argument I can offer is that a rotating magnet does not create a magnetic field

No, but it **does** create an _electric_ field, for example like with a [Bottle dynamo](https://en.wikipedia.org/wiki/Bottle_dynamo):

![Gazelle](https://github.com/a-cosmology-group/acg/assets/135433222/eb964e29-4aab-45e6-8837-d3eeabacb049)

**Reply by Francois-Zinserling** [reply]
@HanDeBruijn that's cheating. :)
A bottle dynamo has several magnets. Not 'homogenous'.

**Comment by genotics** [comment]
The Fay article is available here <https://relativemotionorg.files.wordpress.com/2024/01/on_sciama_1953.pdf>

**Comment by HanDeBruijn** [comment]
When looking only at the last line of [_Universes, Black Holes and Elementary Particles_](http://redshift.vif.com/JournalFiles/Pre2001/V00NO20PDF/Nr20bro.pdf), it is seen that Peter Browne calculates for the Hubble constant a value $H=108\mbox{ }km/s/Mpc$ which is rather deviant form the best current values $H\approx 73.4\mbox{ }km/s/Mpc$ (Riess).
>"Electromagnetic energy density in the external field of an electron provides gravitational mass density which is the source of a gravitational field. The gravitational field, in turn, provides energy density which makes a further contribution to the source of the same field (i.e. a self-source term)."

Yes, it's possible to calculate the self-energy of an electron and via $E=mc^2$ it can be identified with an [Electromagnetic Mass](http://www.alternatievewiskunde.nl/MijnBoek/vanBopp.htm). We all know that such a Mass gravitates. And the gravitational field itself has an (negative) [Self Energy](http://www.alternatievewiskunde.nl/cosmology/nasa.htm).
Furthermore, it's an interesting idea that gravitational mass behaves like imaginary charge, as displayed in equation (33) of the paper. But, as has been remarked by yourself and others, the mathematics is hardly readable. So I gave up.
>From this he deduces that "**A universe, therefore, behaves as a black hole** for both an internal and an external observer. The black hole horizon at r = R severs connection between regions r < R and r > R, in which sense a universe becomes a perfectly isolated system in an infinite Cosmos."

**It's in the air !** For quite some time, apparently. Both @ExpEarth and @HanDeBruijn have picked up this idea of a Black Hole Universe.
>If any members are tempted to investigate the work of Peter and Amitabha as a possible foundation for a more robust cosmological (redshift) model [ .. ]

About the [_Inertial Induction and the Potential Energy Problem_](http://redshift.vif.com/JournalFiles/Pre2001/V02no2PDF/V02N2GHO.PDF). Much of the theory as put forward by Amitabha Ghosh has been worked out further, for example in the section [Origin of Inertia](http://www.alternatievewiskunde.nl/cosmology/Ghosh.htm) at my website.
